packages/website/docs/_components_pages/main/BusyIndicator.mdx

```diff
@@ -24,4 +25,9 @@ The BusyIndicator comes in several sizes - Small, Medium and Large.
 ### Busy component
 The BusyIndicator can be placed over another web component.
 
-<WithComponent />
\ No newline at end of file
+<WithComponent />
+
+### TextPlacement
```

Comment:
Great to see a sample from the ones.

- I think `TextPlacement` written separately is better as title - `Text Placement`

- Some minor typos below - `tex tor` -> `text or`

- Something like this: `The BusyIndicator can display text either above (via placement="Top") or below the animated dots (by default).`